data/data/ibmcloud/network/vpc/dedicated-hosts.tf

```diff
@@ -0,0 +1,70 @@
+locals {
+  dhosts_master_create = [for dhost in var.dedicated_hosts_master : dhost if lookup(dhost, "id", "") == ""]
+  dhosts_worker_create = [for dhost in var.dedicated_hosts_worker : dhost if lookup(dhost, "id", "") == ""]
+  dhosts_master_merged = [
+    for dhost in var.dedicated_hosts_master :
+    lookup(dhost, "id", "") == ""
+    ? ibm_is_dedicated_host.control_plane[index(ibm_is_dedicated_host.control_plane.*.zone, dhost.zone)].id
+    : dhost.id
+  ]
+}
+
+############################################
+# Dedicated hosts (Control Plane)
```

Comment:
Is the vpc module really the proper place for creating the dedicated hosts? At first blush, dedicated hosts are not part of a VPC.

Reply:
You're right there. I just put it there for the sake of ease, but I can create a new module.